src/main/java/seedu/address/model/person/Student.java

```diff
@@ -73,7 +92,7 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder(super.toString());
-
+        builder.append("; ").append(getClassTiming());
```

Comment:
I think you forgot to put "ClassTiming: " in, you can refer to the toString() for Person.java

Reply:
Oh yes, thanks for catching